File: app/src/main/java/Model/Comment.java
package Model;

import com.google.firebase.Timestamp;

public class Comment {

    String UID, commentContent;
    Timestamp date;

    public Comment() {
    }

    public String getUID() {
        return UID;
    }

    public void setUID(String UID) {
        this.UID = UID;
    }

    public String getCommentContent() {
        return commentContent;
    }

    public void setCommentContent(String commentContent) {
        this.commentContent = commentContent;
    }

    public Timestamp getDate() {
        return date;
    }

    public void setDate(Timestamp date) {
        this.date = date;
    }
}

File: data/model/src/main/java/com/harera/model/modelget/Comment.kt
package com.harera.model.modelget

import com.google.firebase.Timestamp
import com.google.firebase.firestore.IgnoreExtraProperties

@IgnoreExtraProperties
class Comment {
    lateinit var uid: String
    lateinit var postId: String
    lateinit var commentId: String
    lateinit var comment: String
    lateinit var time: Timestamp
    lateinit var profileName: String
}
